golang/cmd/kafka-to-postgresql/_pxTemplate.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"database/sql"
+	jsoniter "github.com/json-iterator/go"
+	"go.uber.org/zap"
+)
+
+type TEMPLATE struct{}
+
+type template struct {
+	Scrap       uint32 `json:"scrapCount"`
```

Comment:
Add note "please change content of this template struct when applying" or somethin similar